src/swagger.yaml

```diff
@@ -241,6 +240,28 @@ paths:
                 description: Server Error
                 schema:
                   $ref: '#/definitions/Exception'
+  '/sensor':
+    post:
+      tags:
+        - Add a new sensor
+      summary: Add a new sensor
+      x-swagger-router-controller: controllers.sensor_info_controller
+      operationId: post_new_sensor
+      produces:
+        - application/json
+      consumes:
+      - "application/json"
+  
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Name of the orgnization to which the bearing belongs to"
+        required: true
+        schema:
+          $ref: "#/definitions/PostSensor"
+      responses:
+        405:
+          description: "Invalid input"
```

Comment:
maybe it's still work in progress, but there is no `200` response?

Reply:
Fixed